addons/web/static/src/js/views/list/list_controller.js

```diff
@@ -400,6 +400,25 @@ var ListController = BasicController.extend({
             return result;
         }, []);
         return new Promise((resolve, reject) => {
+            const saveRecords = () => {
+                this.model.saveRecords(this.handle, recordId, validRecordIds, fieldName)
+                    .then(async () => {
+                        this._updateButtons('readonly');
+                        const state = this.model.get(this.handle);
+                        // We need to check the current multi-editable state here
+                        // in case the selection is changed. If there are changes
+                        // and the list was multi-editable, we do not want to select
+                        // the next row.
+                        this.selectedRecords = [];
+                        await this.renderer.updateState(state, {
+                            keepWidths: true,
+                            selectedRecords: [],
+                        });
+                        this.renderer.focusCell(recordId, node);
+                        resolve(!Object.keys(changes).length);
```

Comment:
damn it... it's wrong: saveRecords is called in both cases. Sorry I mislead you... So you can either add an arg to saveRecords (boolean: multi or not), or keep using the isMultipleRecordEdition as you did earlier. That being said, I like the fact that we directly resolve with a boolean now.